.gitignore

```diff
@@ -1,16 +1,10 @@
-# Node generated files
-node_modules
-npm-debug.log
-# OS generated files
-Thumbs.db
-.idea
-.DS_Store
-# Ignored files
-*.js
-*.map
-*.d.ts
-**/*.ngfactory.ts
-**/*.metadata.json
-**/*.ngsummary.json
-.vscode
+$ cat .gitignore
```

Comment:
This seems like a copy paste from the bash which shouldn't be part of the gitignore